packages/core/src/cli/lib/init-helpers.ts

```diff
@@ -515,48 +522,6 @@ export async function initDmnoForService(workspaceInfo: ScannedWorkspaceInfo, se
   // }
 
 
-  // INSTALL KNOWN INTEGRATIONS
```

Comment:
this was happening twice. I had moved it but for forgot to remove the second one